build/media_source/plg_multifactorauth_webauthn/js/webauthn.es6.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Multifactorauth.webauthn
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+((Joomla, document) => {
+  'use strict';
+
+  let authData = null;
+
+  const arrayToBase64String = (a) => btoa(String.fromCharCode(...a));
+
+  const base64url2base64 = (input) => {
+    let output = input
+      .replace(/-/g, '+')
+      .replace(/_/g, '/');
+    const pad = output.length % 4;
+    if (pad) {
+      if (pad === 1) {
+        throw new Error('InvalidLengthError: Input base64url string is the wrong length to determine padding');
+      }
+      output += new Array(5 - pad).join('=');
+    }
+    return output;
+  };
+
+  const displayError = (message) => {
+    try {
+      Joomla.renderMessages({
+        error: message
```

Comment:
```suggestion
        error: message,
```
https://ci.joomla.org/joomla/joomla-cms/54354/1/26